metriq_gym/benchmarks/quantum_volume.py

```diff
@@ -28,11 +28,11 @@ def dispatch_handler(self) -> None:
 
         self.job_manager.add_job(result.to_serializable())
 
-        logging.info(f"Done writing job IDs to file {self.args.jobs_file}.")
+        logging.info(f"Done writing job IDs to file {self.job_manager.jobs_file}.")
```

Comment:
now that jobs_file is not really a user option, we can also remove it from those logging. We can just say "Dispatched job saved", or something on those lines.

Reply:
Yeah, let's obscure this detail for the user. Removed in  89d4547.